src/modules/sub-items/services/bulk.service.js

```diff
@@ -33,12 +33,12 @@ const getBulkMoveRequestOperations = (locations, destination) => {
     return operations;
 };
 
-const processBulkResponse = (locations, successCode, callback, response) => {
+const processBulkResponse = (locations, callback, response) => {
     const { operations } = response.BulkOperationResponse;
     const locationsMatches = Object.entries(operations).reduce(
         (locationsMatches, [locationId, response]) => {
             const respectiveItem = locations.find((location) => location.id === parseInt(locationId, 10));
-            const isSuccess = response.statusCode === successCode;
+            const isSuccess = 200 <= response.statusCode <= 299;
```

Comment:
`200 <= 400 <= 299` returns _true_
```suggestion
            const isSuccess = 200 <= response.statusCode && response.statusCode <= 299;
```